src/gui/mainwindow.h

```diff
@@ -242,6 +178,64 @@ private slots:
     void toolbarTextBeside();
     void toolbarTextUnder();
     void toolbarFollowSystem();
-};
 
+private:
+    QIcon getSystrayIcon() const;
+#ifdef Q_OS_WIN
+    bool addPythonPathToEnv();
+    void installPython();
 #endif
+
+    void dropEvent(QDropEvent *event) override;
+    void dragEnterEvent(QDragEnterEvent *event) override;
+    void closeEvent(QCloseEvent *) override;
+    void showEvent(QShowEvent *) override;
+    bool event(QEvent * event) override;
```

Comment:
Just pointing out that these are `protected` normally. If making them `private` doesn't matter then leave them as is.